2.x-centos7/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER mail@racktear.com
 RUN groupadd tarantool \
     && adduser -g tarantool tarantool
 
-ENV TARANTOOL_VERSION=2.1.2-119-gc118d5142 \
+ENV TARANTOOL_VERSION=2.2.1-3-g878e2a42c \
```

Comment:
Why tarantool version in 2.x-centos7 is not the same as in 2.x? I don't get your approach to choose a version.